core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/data/UserDataBinderImpl.java

```diff
@@ -384,9 +384,6 @@ private boolean isPasswordMapped(final ExternalResource resource) {
 
         AnyUtils anyUtils = anyUtilsFactory.getInstance(AnyTypeKind.USER);
 
-        // realm
```

Comment:
why moving the `setRealm()` method invocation?

Reply:
I described in https://github.com/apache/syncope/pull/289#issuecomment-921725709